rel-eng/tito.props

```diff
@@ -401,6 +401,29 @@ whitelist = ansiblerole-insights-client
   rubygem-smart_proxy_salt
   rubygem-smart_proxy_spacewalk
 
+[foreman-client-nightly-rhel7]
+disttag = .el7
+scl = tfm
+whitelist = katello-host-tools
```

Comment:
Should we keep the whitelists empty as well if we're not adding any packages yet?